src/soar_sdk/abstract.py

```diff
@@ -18,6 +18,10 @@ class SOARClient(ABC):
     def get_soar_base_url(self) -> str:
         pass
 
+    @abstractmethod
+    def get_product_installation_id(self) -> str:
+        pass  # pragma: no cover
```

Comment:
Nit: pretty sure the pragma is unnecessary
```suggestion
        pass
```